app/org/maproulette/models/dal/TaskDAL.scala

```diff
@@ -547,6 +547,49 @@ class TaskDAL @Inject()(override val db: Database,
     }
   }
 
+  /**
```

Comment:
I think this work would better in the TaskController. Instead of calling this function, I think we should call the lockItem and unlockItem directly from the API calls startOnTask and releaseTask.